src/main/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -65,6 +62,9 @@ public class LoginControllerV2 {
 
     private static final String STATE_COOKIE = "state";
 
+    @Value("${find-a-grant.url}")
+    private String findAGrantUrl;
```

Comment:
Renaming this to findBaseUrl or findHostBaseUrl might be more descriptive here?

Reply:
>is there a related terraform change for this?
no